custom/enikshay/nikshay_datamigration/management/commands/create_enikshay_cases.py

```diff
@@ -26,10 +27,17 @@ def handle(self, domain, **options):
         num_succeeded = 0
         num_failed = 0
         logger.info('Starting migration of %d patient cases on domain %s.' % (total, domain))
+        nikshay_codes_to_location = get_nikshay_codes_to_location(domain)
+        nikshay_ids_to_preexisting_nikshay_person_cases = get_nikshay_ids_to_preexisting_nikshay_person_cases(
+            domain
+        )
```

Comment:
you never have the same nikshay ID twice in the same file right? otherwise we'd have to update or invalidate this when we add people.